graylog2-web-interface/src/views/components/WidgetQueryOverride.tsx

```diff
@@ -62,12 +68,18 @@ type Props = {
 
 const WidgetQueryOverride = ({ value, onReset }: Props) => (
   <Wrapper>
-    <TimeRangeInfo>
-      <QueryInput onChange={() => Promise.resolve('')} onExecute={() => {}} disabled value={value.query_string} />
+    <QueryInfo>
+      <StyledQueryInput onChange={() => Promise.resolve('')}
```

Comment:
Maybe it would be better to make `onChange`  and `onExecute` not required in  `QueryInput`